fdbserver/ProxyCommitData.actor.h

```diff
@@ -112,7 +121,28 @@ struct ProxyStats {
 	    commitBatchingWindowSize("CommitBatchingWindowSize",
 	                             id,
 	                             SERVER_KNOBS->LATENCY_METRICS_LOGGING_INTERVAL,
-	                             SERVER_KNOBS->LATENCY_SAMPLE_SIZE) {
+	                             SERVER_KNOBS->LATENCY_SAMPLE_SIZE),
+	    commitBatchQueuingDist(Histogram::getHistogram(LiteralStringRef("CommitProxy"),
+	                                                   LiteralStringRef("commitBatchQueuing"),
```

Comment:
It looks like the existing histograms are capitalizing the first word here. My suggestion would be to stick to a single format for these to avoid any ambiguity, and trace events in general prefer capitalizing first words. That means this would become `CommitBatchQueueing`.

Reply:
Done.